packages/amplication-client/src/Entity/RelatedEntityFieldField.tsx

```diff
@@ -37,7 +44,7 @@ const RelatedEntityFieldField = ({ entityDisplayName }: Props) => {
         <EntityRelationFieldsChart
           fixInPlace={false}
           applicationId={data.entity.appId}
-          entityId={data.entity.id}
+          entityId={entity}
```

Comment:
I dont think this is the right fix for this issue. 
I expected to see the fix in the component `amplication-client/src/Entity/EntityRelationFieldsChart.tsx`
The component includes 4 links, only the first pair is not working correctly - the second one is fine. I do not expect to see a change of the component props, rather just a change in how the component builds the links internally

Reply:
Ok, will do the changes